docs/source/tertiary/pipe-transformer.rst

```diff
@@ -128,4 +128,14 @@ The following options are not supported:
  - ``path`` options are ignored
  - The ``parserLib`` option is ignored. ``univocity`` is always used as the CSV parsing library.
 
+Cleanup
+=======
+
+The pipe transformer performs RDD caching to optimize performance. Spark automatically drops out old data partitions in
+a least-recently-used (LRU) fashion. If you would like to manually clean up the RDDs cached by the pipe transformer
+instead of waiting for them to fall out of the cache, use the pipe cleanup transformer on any DataFrame.
+
```

Comment:
Note that this should be done after the result of the pipe transformer has been written out (or however it's ultimately materialized).